.github/workflows/shell.yml

```
@@ -48,10 +48,15 @@ jobs:
   lint_shfmt:
     name: Shell - lint_shfmt
     runs-on: ubuntu-20.04
-    container:
-      image: ghcr.io/build-trust/ockam-builder@sha256:9ee4558d834514e60a50c41bbf38c6ecae47d94dcfaa4df6a7256c262a7b0f4b
     steps:
       - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
         with:
           ref: ${{ github.event.inputs.commit_sha }}
-      - run: shfmt -d $(find . -not -path "./demos/*" -iname "*.sh" -o -name "*.bash")
+
+      - name: Install Nix
+        uses: cachix/install-nix-action@5cfd5166ea7680b784fd8cf556c6d07a9ecce260
```

```suggestion
        uses: DeterminateSystems/nix-installer-action@3ebd1aebb47f95493b62de6eec0cac3cd74e50a9
```

We want to use the installer that defaults all of the flake settings to be enabled.